test/markup/r/numbers.expect.txt

```diff
@@ -0,0 +1,19 @@
+<span class="hljs-comment"># Numbers</span>
+
+<span class="hljs-number">0</span>, <span class="hljs-number">01</span>, <span class="hljs-number">08</span>, <span class="hljs-number">123456</span>, <span class="hljs-number">1256.701</span>, <span class="hljs-number">123e3</span>, <span class="hljs-number">123E+3</span>, <span class="hljs-number">1.23e-3</span>, <span class="hljs-number">1.23E3</span>, <span class="hljs-number">.25</span>, <span class="hljs-number">2.</span>
+
+<span class="hljs-comment"># Integers</span>
+
+<span class="hljs-number">123L</span>, -<span class="hljs-number">50L</span>
```

Comment:
Is the intention to leave - outside the number intentional?  Just checking.

Reply:
It‘s intentional because that’s what the previous lexer did. It also makes lexing easier because I don’t have to disambiguate `a<-3` and `a< -3` (the former needs to be parsed as `a <- 3`). That said, I can change the rule if desired.

Reply:
It's fine as-is, was just making sure it was an intentional choice.